tests/blobstore_common_tests.py

```diff
@@ -48,6 +48,17 @@ def test_get_content_type(self):
                 self.test_fixtures_bucket,
                 "test_good_source_data_DOES_NOT_EXIST")
 
+    def test_get_creation_date(self):
+        created = self.handle.get_creation_date(
```

Comment:
maybe try creating an object and verify that the creation date is recent?

Reply:
That feels like we're testing GS/S3's logic